avocado/utils/network/interfaces.py

```diff
@@ -314,3 +314,29 @@ def remove_ipaddr(self, ipaddr, netmask):
         except Exception as ex:
             msg = 'Failed to remove ipaddr. {}'.format(ex)
             raise NWException(msg)
+
+    def move_interface_file(self, ignore_missing=True):
```

Comment:
I suggest renaming this to `restore_from_backup()`.

Also, silent operations are not good practices. If the user doesn't mind of that, a good alternative would be to raise a proper exception incase of the backup file does not exists, and the user will handle the exception.